test/jit/test_recursive_script.py

```diff
@@ -159,7 +158,6 @@ def a(x):
             # Make sure that no entries are left over from the previous failure
             FileCheck().check_count("is being compiled", 2).run(str(e))
 
-    @unittest.skipIf(sys.version_info[:2] < (3, 7), "Class annotations are a thing in > 3.5, need to fix for < 3.7")
```

Comment:
I think this comment was out of date? Not totally sure, but this test now passes for python3.6 and python3.7 and checks the functionality of Final